packages/sit-onyx/src/composables/useSkeletonState.ts

```diff
@@ -0,0 +1,83 @@
+import { computed, inject, provide, type ComputedRef, type InjectionKey, type Reactive } from "vue";
+
+const SKELETON_INJECTION_KEY = Symbol() as InjectionKey<
+  ReturnType<typeof createSkeletonInjectionContext>
+>;
+
+/**
+ * Props on the parent component.
+ * These are injected, so that they can be used in any child components.
+ */
+export type SkeletonInjectedProps = {
+  skeleton: boolean;
+};
```

Comment:
```suggestion
 * Prop on the parent component.
 * It's value is provided, so that it can be used in child components.
 */
export type SkeletonProvidedProp = {
  skeleton: boolean;
};
```